mobile-widgets/qmlmanager.cpp

```diff
@@ -1436,14 +1434,9 @@ QStringList QMLManager::buddyInit() const
 	int i = 0;
 	for_each_dive (i, d) {
 		QString temp = d->buddy;
-		if (!temp.isEmpty() && !temp.contains(",")){
+		if (!temp.isEmpty()){
 			buddies << d->buddy;
 		}
```

Comment:
I'm not sure you are correct. d->buddy is a "char *", so by removing this check you now add "Linus, Joe, John" as one completion to the drop down. I'm not saying that's necessarily wrong - it's just a bit confusing if you dive with a lot of different combinations of partners. The existing code instead took the string apart and added each buddy individually. I think :-)